TEST.md

```diff
@@ -0,0 +1,35 @@
+
+# Codacy test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5500053)